src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -282,7 +282,15 @@ class Importer(val output: java.io.PrintWriter) {
         }
 
       case UnionType(left, right) =>
-        TypeRef.Union(typeToScala(left), typeToScala(right))
+        def visit(tpe: TypeTree, visited: List[TypeRef]): List[TypeRef] =
```

Comment:
Please add `{}` to this method body, since it spans more than 1 line. (see https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#braces)

Reply:
Done